src/MailingListBundle/Command/SyncSubscriptionCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Opifer\MailingListBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Opifer\MailingListBundle\Entity\MailingList;
+use Opifer\MailingListBundle\Entity\Subscription;
+
+class SyncSubscriptionCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('sync:subscriptions')
```

Comment:
Can you please prefix the command name with `opifer:`?

So; `opifer:sync:subscriptions`